template.yaml

```diff
@@ -534,6 +534,14 @@ Resources:
               "Resource": "arn:aws:lambda:AWS::REGION:AWS::ACCOUNT_ID:function:SungardAS-Federation",
               "InputPath": "$.federation",
               "ResultPath": "$.credentials",
+              "Retry": [
+                {
+                  "ErrorEquals": ["AccessDenied"],
```

Comment:
Shouldn't we retry for any error?

Reply:
No actually we should retry on the specified error. Because in lambda we get error due to GUID also if it is not provided which shouldn't be the condition to retry.

Reply:
@mayankbharadwaj - your point is very well thought.